src/modules/75-cd/components/PipelineSteps/CloudFormation/CloudFormationCreateStack/CloudFormationCreateStackRef.tsx

```diff
@@ -0,0 +1,650 @@
+                  disabled: readonly
+                }}
+              />
+            </div>
+            <div className={cx(stepCss.formGroup, stepCss.sm)}>
+              <FormMultiTypeDurationField
+                name="timeout"
+                label={getString('pipelineSteps.timeoutLabel')}
+                multiTypeDurationProps={{ enableConfigureOptions: false, expressions, allowableTypes }}
+                disabled={readonly}
+              />
+            </div>
+            <div className={css.divider} />
+            <div className={stepCss.formGroup}>
+              <FormInput.MultiTextInput
+                name="spec.provisionerIdentifier"
+                label={getString('pipelineSteps.provisionerIdentifier')}
+                multiTextInputProps={{ expressions, allowableTypes }}
+                disabled={readonly}
+                className={css.inputWidth}
+              />
+              {getMultiTypeFromValue(values.spec?.provisionerIdentifier) === MultiTypeInputType.RUNTIME && (
+                <ConfigureOptions
+                  value={values.spec?.provisionerIdentifier as string}
+                  type="String"
+                  variableName="spec.provisionerIdentifier"
+                  showRequiredField={false}
+                  showDefaultField={false}
+                  showAdvanced={true}
+                  onChange={value => {
+                    setFieldValue('spec.provisionerIdentifier', value)
+                  }}
+                  isReadonly={readonly}
+                  className={css.inputWidth}
+                />
+              )}
+            </div>
+            <div className={stepCss.formGroup}>
+              <FormMultiTypeConnectorField
+                label={<Text color={Color.GREY_900}>{getString('pipelineSteps.awsConnectorLabel')}</Text>}
+                type={Connectors.AWS}
+                name="spec.configuration.connectorRef"
+                placeholder={getString('select')}
+                accountIdentifier={accountId}
+                projectIdentifier={projectIdentifier}
+                orgIdentifier={orgIdentifier}
+                style={{ marginBottom: 10 }}
+                multiTypeProps={{ expressions, allowableTypes }}
+                disabled={readonly}
+                width={300}
+              />
+            </div>
+            <Layout.Vertical className={css.addMarginBottom}>
+              <Label style={{ color: Color.GREY_900 }} className={css.configLabel}>
+                {getString('regionLabel')}
+              </Label>
+              <Layout.Horizontal className={stepCss.formGroup}>
+                <MultiTypeInput
+                  name="spec.configuration.region"
+                  selectProps={{
+                    addClearBtn: false,
+                    items: regions
+                  }}
+                  disabled={readonly}
+                  onChange={({ value }: any) => setFieldValue('spec.configuration.region', value)}
```

Comment:
Codacy has a fix for the issue: [Unexpected any. Specify a different type.](https://app.codacy.com/gh/harness/harness-core-ui/pullRequest?prid=9232213)
```suggestion
                  onChange={({ value }: unknown) => setFieldValue('spec.configuration.region', value)}
```